### PR TITLE
New translations User guide 10.11.0

### PR DESCRIPTION
**Pull request by MWindel**
Added the newly translated files to the German User guide.

### CODE REVIEW

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md [hunk 1]
```diff
@@ -248,11 +263,11 @@ Platzhalter werden nicht auf Korrektheit geprüft. Jeder nicht erkannte oder fal
...
+Die Integration wurde erfolgreich mit Azure AD getestet. Die Konfigurationsschritte finden Sie unter [https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-oauth-code](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-oauth-code).
 
-Während der Konfiguration der Azure AD entspricht die Redirect-URI Ihrer vollständigen Domain-Adresse. In diesem Dokument verwenden wir beispielhaft "http://documentation.{{< domain-c8y >}}/tenant/oauth". In Azure AD sind keine weiteren Schritte erforderlich.
+Verwenden Sie beim Konfigurieren Ihres Azure AD Ihre vollständige Domain-Adresse als Redirect-URI. In diesem Dokument verwenden wir beispielhaft "http://documentation.{{< domain-c8y >}}/tenant/oauth". Die Redirect-URI muss für eine Webanwendung und nicht für eine Einzelseitenanwendung festgelegt sein. In Azure AD sind keine weiteren Schritte erforderlich.
 
-##### {{< product-c8y-iot >}} Konfiguration
+##### Cumulocity IoT-Konfiguration
```
Comment (MWindel) [inline content/benutzerhandbuch/administration-de-bundle/changing-settings.md:270]: @BeateRixen Should I enter the variable here once more, eventhough it is a heading?

Comment (BeateRixen) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:270]: No, in general variables should not show up in headings.

File: content/benutzerhandbuch/cockpit-de-bundle/dashboards.md [hunk 2]
```diff
@@ -1,27 +1,11 @@
...
-
-  Cumulocity IoT enthält vordefinierte Widget-Typen, z. B. Alarm- oder Datenpunktlisten, lineare Zeiger oder Tachos. Da jeder Widget-Typ andere Daten anzeigt, sind zu seiner Konfiguration unterschiedliche Parameter notwendig. Nähere Informationen zu den einzelnen Widget-Typen und ihrer Konfiguration finden Sie in der Dokumentation der Widgets-Sammlung.
 weight: 40
 ---
 
-
 Dashboards ermöglichen eine individuelle Visualisierung Ihrer Daten anhand verschiedener Widgets. Widgets können Karten, Bilder, Graphen, Tabellen und andere grafische Darstellungen von Daten anzeigen.
 
-{{< product-c8y-iot >}} bietet eine Reihe voreingestellter Widgets Details finden Sie im Abschnitt [Widgets-Sammlung](#widgets-collection). Sie können auch eigene Widgets entwickeln und Ihrem {{< product-c8y-iot >}}-Konto hinzufügen, siehe [Web SDK guide](/web/).
+{{< product-c8y-iot >}} bietet eine Reihe voreingestellter Widgets, Details finden Sie im Abschnitt [Widgets-Sammlung](#widgets-collection). Sie können auch eigene Widgets entwickeln und Ihrem {{< product-c8y-iot >}}-Konto hinzufügen. Weitere Informationen finden Sie im [Web SDK guide](/web/).
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/dashboards.md:8]: Web SDK guide > Web SDK Guide

File: content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md [hunk 3]
```diff
@@ -242,4 +225,4 @@ Messwerte können als CSV- oder Excel-Datei exportiert werden. Die exportierten
 
 Klicken Sie auf **Mehr...** in der oberen Menüleiste und wählen Sie je nach Bedarf entweder **Als CSV herunterladen** oder **Als Excel herunterladen**.
 
-Die Download-Datei wird erstellt, wir im folgenden Fenster angezeigt. Dies kann eine Weile dauern, abhängig von der Anzahl der Datenpunkte im Daten-Explorer. Wenn die Download-Datei fertiggestellt ist, klicken Sie auf **Herunterladen**.
+Die Download-Datei wird erstellt, wir im folgenden Fenster angezeigt. Dies kann eine Weile dauern, abhängig von der Anzahl der Datenpunkte im Daten-Explorer. Wenn die Download-Datei fertiggestellt ist, klicken Sie auf **Herunterladen**.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md:228]: wir im folgenden Fenster angezeigt > wie im folgenden Fenster angezeigt

File: content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
 layout: redirect
+title: Verwenden von SmartREST-Templates
+weight: 80
 ---
 
+Bei den SmartREST-Templates handelt es sich um eine Sammlung von Anfrage- und Antwort-Templates, die verwendet werden, um CSV-Daten und {{< product-c8y-iot >}} Rest API-Aufrufe zu konvertieren. Sie können SmartREST-Templates etwa verwenden, um einfach Geräte zur Plattform hinzuzufügen anstatt jedes Mal manuell eine Anfrage zu schreiben.
+
+Um ein neues Template hinzuzufügen, importieren Sie entweder ein vorhandenes Template oder erstellen Sie ein neues.
+Um ein Template zu importieren, klicken Sie rechts in der oberen Menüleiste auf **Template importieren**, wählen Sie eine hochzuladende Datei aus und geben Sie einen Namen und eine eindeutige ID ein.
+Um ein neues Template zu erstellen, klicken Sie rechts in der oberen Menüleiste auf **Template erstellen**, geben Sie einen Namen und eine eindeutige ID ein und fügen Sie die erforderlichen Nachrichten oder Antworten hinzu, wie unter *Device Management > SmartREST-Templates* im *User guide* beschrieben.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md:11]: *Device Management > SmartREST-Templates*  > sThis should better be an active link. Must be modified in the English file as well.

File: content/benutzerhandbuch/cockpit-de-bundle/dashboards.md (hunk 2 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/dashboards.md:8]: Done

File: content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md (hunk 3 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/data-explorer.md:228]: Done

**Review by rssag — APPROVED** [review]

File: content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md (hunk 4 above)
Comment (MWindel) [reply content/benutzerhandbuch/device-management-de-bundle/smartrest-templates.md:11]: Done

File: content/benutzerhandbuch/administration-de-bundle/changing-settings.md (hunk 1 above)
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/changing-settings.md:270]: Ok, I left it as is

**Review by rssag — DISMISSED** [review]

**Review by BeateRixen — DISMISSED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**